src/services/ipfs-service.ts

```diff
@@ -16,7 +16,7 @@ const { serialize, MsgType } = PubsubMessage
 
 const DEFAULT_GET_TIMEOUT = 30000 // 30 seconds
 const MAX_CACHE_ENTRIES = 100
-const IPFS_PUT_TIMEOUT = 60 * 1000 // 60 seconds
+const IPFS_PUT_TIMEOUT = 120 * 1000 // 2 minutes
```

Comment:
do you have any idea *why* it's taking 2 full minutes just to put a record into IPFS?

I'm actually kind of confused on what the expected latency is from ipfs puts.  I guess I would expect it to be comparable to reads, since you need to walk the DHT to find the peers with the closest XOR distance to the CID to register with them as providers.  But empirically, I've generally seen IPFS puts be extremely fast, far faster than gets.  Any idea what the expected behavior here is?  @oed @smrz2001 @zachferland?

Reply:
This is very strange yes. the "dag put" api should not wait for the DHT providing to finish afaik.

Reply:
If you try doing `echo '{ "test": 123 }' | ipfs dag put` on a local ipfs node it is instant!

Reply:
I guess the core of this change is adding a call to `pin` so I guess the real extra time is coming from the pin not the put, but that makes even less sense to take a long time to me.  I'd think pinning a CID is just making a small update to the local node's data files to mark that it shouldn't be garbage collected.  Should be super fast

Reply:
Is this much time consistently needed locally and in tests as well, or was this updated because of something we observed on our infra?

On infra, it could be a variety of reasons, so I wouldn't update the default to deal with that.

Reply:
Doing `echo '{ "test": 123 }' | ipfs dag put --pin` is also instant on my machine.

Reply:
oh wait, recursive pins?
Not sure if that's enabled by default.

Reply:
I haven't had timeout issues locally, only in CI, but now I have another issue preventing from running tests locally:
```
 src/app.ts:24:25 - error TS2307: Cannot find module './version.js' or its corresponding type declarations.

 24 import { version } from './version.js'
```

Reply:
Other test suites run but I get `TypeError: fetch failed` with the `ceramic_integration` test locally so I can't check if the timeouts happen locally as well.